typescript/sdk/src/ism/HyperlaneIsmFactory.ts

```diff
@@ -230,6 +238,28 @@ export class HyperlaneIsmFactory extends HyperlaneApp<ProxyFactoryFactories> {
     return contract;
   }
 
+  protected async deployCCIPIsm(
+    destination: ChainName,
+    config: CCIPIsmConfig,
+  ): Promise<CCIPIsm> {
```

Comment:
the CCIP hook/ism deploy logic has been moved to the HyperlaneCCIPDeployer in the next PR https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/5412